src/helpers/paginationHelper.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @description class representing pagination helper functions
+ * @class paginationHelper
+ */
+class paginationHelper {
+  /**
+   * @description - This method is responsible for returning numbeer of article per page
+   * @static
+   * @param {number} number - parameter sent to controller for get all article
+   * @returns {number} - object representing response message
+   * @memberof paginationHelper
+   */
+  static numberOfArticlePerPage(number) {
+    return number;
+  }
+
+  /**
+   ** @description - This method is responsible for verifying if a query parameter is negative
+   * @param {*} request
+   * @param {*} response
+   * @param {*} next
+   * @returns {object} - object representing response message
+   */
+  static checkQueryparameter(request, response, next) {
+    const { page } = request.query;
+    if (page < 1) {
+      return response.status(400).json({
```

Comment:
Hello Emeka, I like your implementation and how readable your code is but I have a feedback. There is a helper function in the helper folder that would help display error messages properly. You can find that [here](https://github.com/andela/balder-ah-backend/blob/staging/src/helpers/index.js).

Reply:
Resolved.